src/Tracker/Tracker.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
```

Comment:
I would add a semicolon at the end of the line just to keep it consistent with App.js